airflow/www/views.py

```diff
@@ -694,110 +694,111 @@ def rendered(self):
             form=form,
             title=title,)
 
+    def _get_log(self, ti, log_filename):
+        """
+        Get log for a specific try number.
+        :param ti: current task instance
+        :param log_filename: relative filename to fetch the log
+        """
+        # TODO: This is not the best practice. Log handler and
+        # reader should be configurable and separated from the
+        # frontend. The new airflow logging design is in progress.
+        log = ''
+        # Load remote log
+        remote_log_base = conf.get('core', 'REMOTE_BASE_LOG_FOLDER')
+        remote_log_loaded = False
+        if remote_log_base:
+            remote_log_path = os.path.join(remote_log_base, log_filename)
+            remote_log = ""
+
+            # S3
+            if remote_log_path.startswith('s3:/'):
+                s3_log = log_utils.S3Log()
+                if s3_log.exists(remote_log_path):
```

Comment:
I think the logs *should* be in s3 here at this point of the code though right? Falling back to non-local logs might not be great, i.e. if logs should have come from a remote location and the logs don't exist there we shouldn't hide that error by showing the local logs.